modules/ROOT/pages/installing-an-enterprise-license.adoc

```diff
@@ -40,15 +40,13 @@ Sometimes the license installation fails and it might be necessary to manually d
 
 == Download your License Key File
 
-* Log in to https://help.mulesoft.com[the Support portal]. If you do not have credentials to log in, please contact your Customer Success Manager.
+. Log in to https://help.mulesoft.com[the Support portal^] using your login information. If you do not have credentials to log in, please contact your Customer Success Manager.
+. Click on the *Support* tab located on the navigation menu of the MuleSoft Help Center home page.
```

Comment:
Minor, but the standard is typically to use just "Click" vs. "Click on". (Throughout)